PostgreSQL build fails missing readline

In a docker build environment, I get this error in 1.0.0 branch when bulding postgres:

```
precision=standard -O2 -I. -I. -I../../../src/interfaces/libpq -I../../../src/include -D_GNU_SOURCE  -I../readline-master/out/include  -c -o command.o command.c
In file included from command.c:45:0:
input.h:21:31: fatal error: readline/readline.h: No such file or directory
 #include <readline/readline.h>
                               ^
```

--- a/build.sh
+++ b/build.sh
@@ -98,8 +98,8 @@ if [ ! -f "$POSTGRESQL_DIR/$POSTGRESQL_OUT/bin/psql" ]; then
 
 	# Configures make for libreadline7 on linux, without for Darwin
 	if [ ! "$(uname -s)" == "Darwin" ]; then
-		exec_cmd "./configure --prefix=$(pwd)/$POSTGRESQL_OUT --with-libraries=../$LIBREADLINE_DIR/out/lib --with-includes=../$LIBREADLINE_DIR/out/include"
-		exec_cmd "LIBRARY_PATH=$PWD/../readline-master/out/lib/ make"
+		exec_cmd "./configure --prefix=$(pwd)/$POSTGRESQL_OUT --with-libraries=$(pwd)/../$LIBREADLINE_DIR/out/lib --with-includes=$(pwd)/../$LIBREADLINE_DIR/out/include"
+		exec_cmd "make"
 	else
 		exec_cmd "./configure --prefix=$(pwd)/$POSTGRESQL_OUT"
 	fi

